### PR TITLE
Fixed a code example and a spelling error

### PR DESCRIPTION
**Pull request by jntn**
Was reading through your barracks documentation and found two small errors.

Good work on choo and barracks by the way! 🙌

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -241,7 +241,7 @@ app.model({
   subscriptions: {
     emitWoofs: (send) => {
       // emit a woof every second
-      setInterval(() =>  send('print', { woof: 'meow?' }), 1000)
+      setInterval(() =>  send('print', { printWoofs: 'meow?' }), 1000)
```
Comment (yoshuawuyts) [inline README.md]: this won't fly because line `248` expects a `"woof"` property to exist on the object

**Comment by yoshuawuyts** [comment]
Oooh, nice, thanks! :D

File: README.md (hunk 1 above)
Comment (jntn) [inline README.md]: Sorry, changed the wrong thing! 😓 It should be `setInterval(() =>  send('printWoofs', { woof: 'meow?' }), 1000)`

The argument in send should match the effect name, right?

Comment (yoshuawuyts) [inline README.md]: yup, exactly! - I like the name "printwoofs"

**Comment by jntn** [comment]
Updated the code example

**Comment by yoshuawuyts** [comment]
Whoops, totes forgot to merge this - fixed in 3b6ef39. Thanks heaps! :sparkles:

**Pull request closed without merging**